tests/unit_tests/test_data_types.py

```diff
@@ -1219,6 +1318,24 @@ def test_object3d_seq_to_json(mock_run, point_cloud_1, assets_path):
     assert obj["count"] == 3
 
 
+def test_object3d_label_is_optional(mock_run):
```

Comment:
I'm not clear if these add any value, other than showing that the types are correct (which is a questionable thing to test, but perhaps helpful as an SDK example, and serves as a reminder that the types have to support these scenarios)

I also tried to do: 
```
box_with_label: "Box3D" = {
        "corners": [],
        "label": "i am a label",
        "color": [0, 0, 0],
    }
```

but that threw an error that I couldn't import that name (presumably due to `if TYPE_CHECKING`

anyway! put this here as a placeholder for the fact that I was thinking about score tests.